src/manager/msManager.js

```diff
@@ -41,5 +42,25 @@ var MsManager = class MsManager {
             }
         });
     }
+
+    removeAll(subject) {
```

Comment:
MsManager is not only a signal Manager. This function should have a more specific name like **RemoveAllSignalsBindTo**

Reply:
OK